compiler/src/dotty/tools/backend/jvm/DottyBackendInterface.scala

```diff
@@ -828,8 +828,8 @@ class DottyBackendInterface(outputDirectory: AbstractFile, val superCallsMap: Ma
     def freshLocal(cunit: CompilationUnit, name: String, tpe: Type, pos: Position, flags: Flags): Symbol =
```

Comment:
`SymUtils` is not used much, to this commit looks smallish. But we should use it more often because it gives us a way to add methods without overloading SymDenotation. In that case I think it is essential that it works for `Symbol` and `SymDenotation` in the same way. So my vote is to leave this as is.